src/nested_pandas/nestedframe/core.py

```diff
@@ -596,7 +596,7 @@ def from_lists(cls, df, base_columns=None, list_columns=None, name="nested"):
 
         # join the nested column to the base_column df
```

Comment:
```suggestion
        # concat the nested column to the base_column df
```
(or even simply delete the comment)